_includes/icons.html

```diff
@@ -55,4 +55,16 @@
           )">
     <path fill-rule="nonzero" clip-rule="nonzero" d="M 21.896770477294922 13 C 22.796770095825195 6 17.29686737060547 0 10.996868133544922 0 C 4.696867942810059 0 -0.8030341863632202 6 0.09696576744318008 13 L 8.996868133544922 13 L 8.996868133544922 17.5 C 8.996868133544922 20 10.996868133544922 22 13.496868133544922 22 C 15.996868133544922 22 17.996868133544922 20 17.996868133544922 17.5 L 17.996868133544922 16 L 14.996868133544922 16 L 14.996868133544922 17.5 C 14.996868133544922 18.299999237060547 14.296868324279785 19 13.496868133544922 19 C 12.696867942810059 19 11.996868133544922 18.299999237060547 11.996868133544922 17.5 L 11.996868133544922 13 L 21.896770477294922 13 Z M 10.996868133544922 3 C 14.596868515014648 3 18.096771240234375 5.900000095367432 18.896770477294922 10 L 3.197063446044922 10 C 3.8970634937286377 5.900000095367432 7.3968682289123535 3 10.996868133544922 3 Z" fill="currentColor"/></g>
   </symbol>
+
+  <symbol id="icon-slack" viewBox="0 0 270 270" style="enable-background:new 0 0 270 270;">
```

Comment:
I had to change the viewbox in the original image to get rid of the huge border. Did you fix that another way?